Makefile

```diff
@@ -47,7 +47,7 @@ manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and Cust
 		output:rbac:artifacts:config=manifests/v2/base/rbac \
 		output:webhook:artifacts:config=manifests/v2/base/webhook
 
-generate: controller-gen ## Generate apidoc, sdk and code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations.
+generate: controller-gen manifests ## Generate apidoc, sdk and code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations.
```

Comment:
Could you remove this?
https://github.com/kubeflow/training-operator/blob/7c5ea70f10d07732826907e75d1e5a50db97a059/.github/workflows/test-go.yaml#L32-L35

Reply:
@tenzen-y Oh, you want to separate it.
Don't we want to include `manifests` command into generate script ?

Reply:
I wanted to say removing "Check manifests" from workflow because we will perform manifests verification with `make generate` since this PR.

Reply:
Oh, you are right!